decred/tests/unit/dcr/test_txscript.py

```diff
@@ -2352,30 +2352,84 @@ def test_scriptNumBytes(self):
             )
 
 
-class TestVSP(unittest.TestCase):
-    def test_blob(self):
-        pi = vsp.PurchaseInfo(
-            addr="someaddr",
-            fees=1.3,
-            script=ByteArray("0a1b2c"),
-            ticketAddr="someotheraddr",
-            vBits=2,
-            vBitsVer=1,
-            stamp=123456,
-        )
-        stakePool = vsp.VotingServiceProvider(
-            "http://localhost:54321", "somerandomkey", mainnet.Name, pi
-        )
-        b = stakePool.serialize()
-        rePool = vsp.VotingServiceProvider.unblob(b.b)
-        self.assertEqual(stakePool.url, rePool.url)
-        self.assertEqual(stakePool.apiKey, rePool.apiKey)
-        self.assertEqual(stakePool.net.Name, rePool.net.Name)
-        rePI = rePool.purchaseInfo
-        self.assertEqual(pi.poolAddress, rePI.poolAddress)
-        self.assertEqual(pi.poolFees, rePI.poolFees)
-        self.assertEqual(pi.script, rePI.script)
-        self.assertEqual(pi.ticketAddress, rePI.ticketAddress)
-        self.assertEqual(pi.voteBits, rePI.voteBits)
-        self.assertEqual(pi.voteBitsVersion, rePI.voteBitsVersion)
-        self.assertEqual(pi.unixTimestamp, rePI.unixTimestamp)
+def test_is_dust_amount():
+    class test:
```

Comment:
Class name should begin with a capital letter, even if they are inner classes like this one.

Having said that, is it worth defining a class for test data? A dict would probably be enough. The data description in the docstring may become a comment.